Створено розмітку мобільного меню.

AngelaKoshel opened the pull request.

AngelaKoshel committed:
Створено розмітку мобільного меню.

OlehKhv commented on src/partials/mobile-menu.html:
```diff
@@ -0,0 +1,22 @@
+<div class="mobile-menu" data-menu>
+  <div class="mobile-container">
+    <svg class="mobile-logo-icon">
```
лого обгорнути в посилання

OlehKhv commented on src/partials/mobile-menu.html:
```diff
@@ -0,0 +1,22 @@
+    </button>
+  </div>
+  <div class="mobile-navigation">
+    <ul class="mobile-item">
```
обгорнути в нав

AngelaKoshel committed:
mobile-logo-icon і mobile-navigation

mobile-logo-icon  обгорнула у тег посилання. mobile-navigation обгорнула у тег nav

OlehKhv commented on src/partials/mobile-menu.html:
```diff
@@ -0,0 +1,26 @@
+<div class="mobile-menu" data-menu>
+  <div class="mobile-container">
+    <a href="">
+      <svg class="mobile-logo-icon">
```
лого вставте через img як я писав в тг

OlehKhv commented on src/partials/mobile-menu.html:
```diff
@@ -0,0 +1,26 @@
+    <nav class="mobile-navigation">
+      <ul class="mobile-item">
+        <li class="mobile-list">
+          <a href="">About us</a>
```
в href додайте якірні посилання `href="#catalog"` і т.д.

OlehKhv replied:
ось тут виправте

AngelaKoshel committed:
Змінила <svg> на <img> та прибрала <div> бо він там не потрібен

Змінила <svg> на <img> та прибрала <div> бо він там не потрібен
